src/test/java/io/cdap/plugin/batch/source/ftp/FTPBatchSourceTest.java

```diff
@@ -82,8 +82,7 @@ public static void ftpTeardown() {
 
   @Test
   public void testSchemaDetection() {
-    String path = String.format("ftp://user2:%s@%s:%d/tmp/file1.txt",
-                                PASSWORD_WITH_SPECIAL_CHARACTERS, HOST, ftpServerPort);
+    String path = String.format("ftp://user:password@%s:%d/tmp/file1.txt", HOST, ftpServerPort);
```

Comment:
had to change this because Hadoop's FTPFileSystem does not work if there is a `:` in the password (SFTP doesn't have this problem)